python/hail/methods/family_methods.py

```diff
@@ -825,14 +824,14 @@ def solve(p_de_novo):
     tm = tm.annotate_entries(__call=de_novo_call)
     tm = tm.filter_entries(hl.is_defined(tm.__call))
     entries = tm.entries()
-    return entries.select('locus',
-                          'alleles',
-                          '__site_freq',
-                          'proband',
-                          'father',
-                          'mother',
-                          'proband_entry',
-                          'father_entry',
-                          'mother_entry',
-                          'is_female',
-                          **entries.__call).rename({'__site_freq': 'prior'})
+    entries = entries.key_by('locus', 'alleles')
```

Comment:
I think the key is already [locus, alleles, proband]. We can probably leave these as is.

Reply:
Corollary: we should document this better in the method 😄